spec/declarations.md

```diff
@@ -10,9 +10,10 @@
 ## Syntax
 
 <x><pre>
-**Declaration**         ::= StandardDeclaration | CustomDeclaration
-**StandardDeclaration** ::= [InterpolatedIdentifier]¹ ':' (Value | Value? '{' Statements '}')
-**CustomDeclaration**   ::= [InterpolatedIdentifier]² ':' InterpolatedDeclarationValue
+**Declaration**              ::= StandardDeclaration | CustomDeclaration
+**StandardDeclaration**      ::= [InterpolatedIdentifier]¹ ':' (Value | Value? Block )
+**CustomDeclaration**        ::= [InterpolatedIdentifier]² ':' InterpolatedDeclarationValue
+InterpolatedDeclarationValue ::= (Interpolation | String)+
```

Comment:
I don't think this definition is right—specifically, "String" doesn't really encapsulate the fact that _almost any_ token is allowed here. It's certainly not just things that parse as unquoted or quoted strings in Sass. If this definition will be relevant for the proposal, it's probably necessary to specifically enumerate the tokens it can contain.

Also, style nit: the left-hand side of syntax definitions should always be bold.

```suggestion
**InterpolatedDeclarationValue** ::= (Interpolation | String)+
```